service/templates/projects.html

```diff
@@ -10,6 +10,7 @@
   window.isSuperUser = false;
   {{end}}
   window.canCreate = {{.CanCreate}};
+  window.validDefaultLoggers = {{.ValidDefaultLoggers}};
```

Comment:
Do you really need to get this from the server? I feel like we can hardcode this in the client

Reply:
I guess not? It just seems better than repeating information in multiple places

Reply:
The source for the data was specifically created for this though. I don't think it's a big deal to repeat this twice, but if you wanted to do that I would put the source in a js file